paddle/fluid/operators/mkldnn/layer_norm_mkldnn_op.cc

```diff
@@ -37,37 +37,22 @@ class LayerNormOneDNNHandler
             engine, cpu_place) {
     const auto fwd_prop_kind = is_test ? dnnl::prop_kind::forward_inference
                                        : dnnl::prop_kind::forward_training;
+
```

Comment:
A random place:
Will these oneDNN kernels in fluid/operators/mkldnn dir be migrated to phi/kernels/onednn dir? Are we doing this task?

Reply:
Hi, oneDNN kernels migration to PHI is finished. Kernels which are still in fluid are left on purpose (description: https://github.com/PaddlePaddle/Paddle/issues/45308)

Reply:
Got it, thanks for your explanation